### PR TITLE
[IMP] make minimum version of Python a variable

### PR DESCRIPTION
**Pull request by oomsveta**
Make the mininum version of Python a variable so that it can be easily retrieved and used by the code. For example, it could be used to detect when a monkey patch should be removed.

### CODE REVIEW

**Comment by oomsveta** [comment]
@odoo/rd-security Hello, could you override ci/security?
import of the sys module to retrieve the current version of python

**Comment by mart-e** [comment]
Do we need it for something that we are changing this in 15.0 and not master?

**Comment by oomsveta** [comment]
I need it for a test in 16

File: odoo/__init__.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
 ]
 
 import sys
-assert sys.version_info > (3, 7), "Outdated python version detected, Odoo requires Python >= 3.7 to run."
+min_py_version = (3, 7)
+assert sys.version_info > min_py_version, f"Outdated python version detected, Odoo requires Python >= {'.'.join(map(str, min_py_version))} to run."
```
Comment (mart-e) [inline odoo/__init__.py:20]: nitpicking
```suggestion
MIN_PY_VERSION = (3, 7)
assert sys.version_info > MIN_PY_VERSION, f"Outdated python version detected, Odoo requires Python >= {MIN_PY_VERSION} to run."
```
(if you are not able to start your server, how is formated the version is not that important

**Review by mart-e — COMMENTED** [review]
@robodoo override=ci/security
Reason: access python version

**Comment by oomsveta** [comment]
robodoo r+

**Pull request closed without merging**